wlauto/common/android/device.py

```diff
@@ -670,6 +670,19 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_network_connected(self):
+            network_host = 'www.google.com'
```

Comment:
As for a dedicated test domain there isn't one that I know off. Most of the examples I've seen use google as a default.